ckanext/datavicmain/iar_ckan_dataset.yaml

```diff
@@ -367,6 +367,11 @@ dataset_fields:
   form_snippet: vic_hidden.html
   validators: ignore_missing
 
+- field_name: nominated_view_id
+  display_snippet: null
+  form_snippet: vic_hidden.html
+  required: false
```

Comment:
I think that `false` is the default value for the `required` flag. Can you check it?